### PR TITLE
Add Attentive

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Attentive is a personalized mobile messaging platform that helps businesses engage with customers through targeted SMS and email marketing. The platform offers tools for automating marketing campaigns, improving customer retention, and driving sales via personalized communication. Attentive provides 2FA for the account login verification process. The website is ranked 132,378 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=attentive.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Comment by fletcherdodd** [comment]
attentive com
I found this SVG that would be better to use instead of the PNG

**Comment by Yang-Nankai** [comment]
Thank you so much, I've been looking for it for a long time and can't find it! @fletcherdodd

File: img/a/attentive.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 224.5 224.5"><circle cx="112.3" cy="112.3" r="112.3" fill="#fff383"/><path fill="#1a1e22" fill-rule="evenodd" d="M133.2 68.5 168 134c1.5 2.7 2.4 5.7 2.7 8.7a23.1 23.1 0 0 1-.8 9.1c-.9 3-2.4 5.7-4.3 8a23.1 23.1 0 0 1-7 5.8 23 23 0 0 1-8.7 2.6 23.1 23.1 0 0 1-9-1 23.3 23.3 0 0 1-13.8-11.4l-25.5-48A17.8 17.8 0 0 0 97 102a18 18 0 0 0-21.4-.5 1 1 0 0 1-.8.3 1 1 0 0 1-.7-.3 1 1 0 0 1-.4-.7 1 1 0 0 1 .3-.8l18.8-32.2c2-3.5 5-6.5 8.7-8.5a23.2 23.2 0 0 1 23.4.5c3.6 2.1 6.4 5.2 8.4 8.8zm-54 99.6c5 0 9.9-1.5 14-4.3a25.5 25.5 0 0 0 9.5-11.4A25.5 25.5 0 0 0 69.4 119a25.5 25.5 0 0 0-11.4 9.4 25.5 25.5 0 0 0 3.1 32.1 25.4 25.4 0 0 0 18 7.5"/></svg>
```
Comment (fletcherdodd) [inline img/a/attentive.com.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 224.5 224.5"><circle cx="112.3" cy="112.3" r="112.3" fill="#fff383"/><path fill="#1a1e22" d="M133.2 68.5 168 134c1.5 2.7 2.4 5.7 2.7 8.7a23.1 23.1 0 0 1-.8 9.1c-.9 3-2.4 5.7-4.3 8a23.1 23.1 0 0 1-7 5.8 23 23 0 0 1-8.7 2.6 23.1 23.1 0 0 1-9-1 23.3 23.3 0 0 1-13.8-11.4l-25.5-48A17.8 17.8 0 0 0 97 102a18 18 0 0 0-21.4-.5 1 1 0 0 1-.8.3 1 1 0 0 1-.7-.3 1 1 0 0 1-.4-.7 1 1 0 0 1 .3-.8l18.8-32.2c2-3.5 5-6.5 8.7-8.5a23.2 23.2 0 0 1 23.4.5c3.6 2.1 6.4 5.2 8.4 8.8zm-54 99.6c5 0 9.9-1.5 14-4.3a25.5 25.5 0 0 0 9.5-11.4A25.5 25.5 0 0 0 69.4 119a25.5 25.5 0 0 0-11.4 9.4 25.5 25.5 0 0 0 3.1 32.1 25.4 25.4 0 0 0 18 7.5"/></svg>
```
Apologies, I missed this part in the optimisation.

Comment (fletcherdodd) [inline img/a/attentive.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 224.5 224.5"><circle cx="112.3" cy="112.3" r="112.3" fill="#fff383"/><path fill="#1a1e22" d="M133.2 68.5 168 134c1.5 2.7 2.4 5.7 2.7 8.7a23.1 23.1 0 0 1-.8 9.1c-.9 3-2.4 5.7-4.3 8a23.1 23.1 0 0 1-7 5.8 23 23 0 0 1-8.7 2.6 23.1 23.1 0 0 1-9-1 23.3 23.3 0 0 1-13.8-11.4l-25.5-48A17.8 17.8 0 0 0 97 102a18 18 0 0 0-21.4-.5 1 1 0 0 1-.8.3 1 1 0 0 1-.7-.3 1 1 0 0 1-.4-.7 1 1 0 0 1 .3-.8l18.8-32.2c2-3.5 5-6.5 8.7-8.5a23.2 23.2 0 0 1 23.4.5c3.6 2.1 6.4 5.2 8.4 8.8zm-54 99.6c5 0 9.9-1.5 14-4.3a25.5 25.5 0 0 0 9.5-11.4A25.5 25.5 0 0 0 69.4 119a25.5 25.5 0 0 0-11.4 9.4 25.5 25.5 0 0 0 3.1 32.1 25.4 25.4 0 0 0 18 7.5"/></svg>
```
Apologies, I missed this part in the optimisation.

**Review by fletcherdodd — CHANGES_REQUESTED** [review]

**Review by fletcherdodd — APPROVED** [review]

**Pull request merged**